Tests/UtilityTests/ProcessSetTests.swift

```diff
@@ -70,7 +69,7 @@ class ProcessSetTests: XCTestCase {
                 threadStartCondition.wait()
             }
         }
-        processSet.terminate()
+        try processSet.terminate()
 
         t.join()
```

Comment:
Did you see time taken by the test `testSigInt` before and after waitUntilAfter() in process set? There should be a noticeable difference I think.